vuln/npm/398.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "398",
+  "title": "Cross-site Scripting (XSS) - Stored",
+  "overview": "[metascrapper] Stored XSS in Open Graph meta properties read by metascrapper",
+  "created_at": "2018-01-25",
+  "updated_at": "2018-03-28",
+  "publish_date": "2018-03-28",
+  "author": "bl4de (https://twitter.com/_bl4de)",
+  "module_name": "metascrapper",
+  "cves": [
+    ""
+  ],
+  "vulnerable_versions": "<=3.9.2",
+  "patched_versions": "",
+  "slug": "metascrapper-cross-site-scripting-(xss)---stored",
+  "recommendation": "No fix is currently available for this vulnerability.\n\nIt is our recommendation to not install or use this module at this time.",
+  "references": "- https://www.hackerone.com/reports/309367",
```

Comment:
The link is incorrect, it has to be without `www.` prefix.
`www.` is an isolated subdomain hosting the landing page and other more complex and less sensitive stuff.
There is a redirect, but it displays an error first.

Reply:
Ref: #199